Unreal/SystemLocalized/det/UnrealShare.det

```diff
@@ -1448,7 +1448,7 @@ ModeString[1]="Klassisch"
 ; EN: DeathMessage="%o was perforated by %k's %w."
```

Comment:
Yet another. Remarks:

Line 1387: "Ist Waffen bleiben aktiviert" → "Ist Waffen Bleiben aktiviert" to make clear this refers to the option that was written in capitals.

Line 1426: "Selbstmord." → "Todeswunsch." if you wish to stick closer to the original. It stresses the masochistic intention of the original phrasing much better I think.

Line 1439: "Gleichgewichtsmodus: " → "Balancing-Art: " I would use the English balancing term here to clearly refer to the game balancing as it's commonly used in gamer jargon. Furthermore it's more obvious this is about gameplay changes at first glance that way, rather than about any sort of physical balance.

Reply:
@SlyMk3 I find "Selbstmord." conveys the original intention very well and would be way more likely to be said by a real person than "Todeswunsch".